src/last10.c

```diff
@@ -28,49 +28,44 @@
 #include "get_time.h"
 #include "globalvars.h"		// Includes glib.h and tlf.h
 
-
 int last10(void) {
```

Comment:
Not related to this PR: `last10()` considers only the hour+minute,  +/- 1 day. So opening an older log file sometimes shows last10 as if it they would be recent QSOs.

Reply:
Yes, adding the date to the calculation would be better.